src/components/SQForm/useForm.tsx

```diff
@@ -14,15 +48,15 @@ function _handleError(name, isRequired) {
   }
 }
 
-function _getIsFulfilled(hasValue, isError) {
+function _getIsFulfilled(hasValue: boolean, isError: boolean) {
   if (hasValue && !isError) {
     return true;
   }
 
   return false;
 }
 
-function _getHasValue(meta) {
+function _getHasValue(meta: any) {
```

Comment:
@20BBrown14 looks like its throwing another any complaint here

Reply:
Does anybody know when these start showing up? I pushed my commit and waited for the build to finish, and a few more minutes and saw no issues here on github. But now they're there so just curious what triggers them.

Reply:
This comes from the `ci` GH Action I wrote.
GH has a neat feature where it will annotate the Files Changed tab with these warnings.
How long it takes for their UI to shows them? I have no idea.